production/loki-mixin/dashboards/loki-reads.libsonnet

```diff
@@ -31,10 +31,9 @@ local utils = import 'mixin-utils/utils.libsonnet';
                          local cfg = self,
 
                          showMultiCluster:: true,
-                         clusterLabel:: $._config.per_cluster_label,
```

Comment:
same here and others

Reply:
Same as above, I did not see it being used